docs/index.rst

```diff
@@ -138,6 +138,7 @@ Variable Discretisation: Discretisers
 - :doc:`api_doc/discretisation/EqualFrequencyDiscretiser`: sorts variable into equal frequency intervals
 - :doc:`api_doc/discretisation/EqualWidthDiscretiser`: sorts variable into equal width intervals
 - :doc:`api_doc/discretisation/DecisionTreeDiscretiser`: uses decision trees to create finite variables
+- :doc:`api_doc/discretisation/IncreasingWidthDiscretiser`: sorts variable into increasing width intervals
```

Comment:
following the slides you linked to the class, would it be not better to say: `sorts variable into geometrical intervals`?